states/MT/governor.yaml

```diff
@@ -0,0 +1,118 @@
+            "Arizona": "Arizona"
+            "Arkansas": "Arkansas"
+            "California": "California"
+            "Colorado": "Colorado"
+            "Connecticut": "Connecticut"
+            "Delaware": "Delaware"
+            "District of Columbia": "District of Columbia"
+            "Federated States of Micronesia": "Federated States of Micronesia"
+            "Florida": "Florida"
+            "Georgia": "Georgia"
+            "Guam": "Guam"
+            "Hawaii": "Hawaii"
+            "Idaho": "Idaho"
+            "Illinois": "Illinois"
+            "Indiana": "Indiana"
+            "Iowa": "Iowa"
+            "Kansas": "Kansas"
+            "Kentucky": "Kentucky"
+            "Louisiana": "Louisiana"
+            "Maine": "Maine"
+            "Marshall Islands": "Marshall Islands"
+            "Maryland": "Maryland"
+            "Massachusetts": "Massachusetts"
+            "Michigan": "Michigan"
+            "Minnesota": "Minnesota"
+            "Minor Outlying Islands": "Minor Outlying Islands"
+            "Mississippi": "Mississippi"
+            "Missouri": "Missouri"
+            "Montana": "Montana"
+            "Nebraska": "Nebraska"
+            "Nevada": "Nevada"
+            "New Hampshire": "New Hampshire"
+            "New Jersey": "New Jersey"
+            "New Mexico": "New Mexico"
+            "New York": "New York"
+            "North Carolina": "North Carolina"
+            "North Dakota": "North Dakota"
+            "Northern Mariana Islands": "Northern Mariana Islands"
+            "Ohio": "Ohio"
+            "Oklahoma": "Oklahoma"
+            "Oregon": "Oregon"
+            "Palau": "Palau"
+            "Pennsylvania": "Pennsylvania"
+            "Puerto Rico": "Puerto Rico"
+            "Rhode Island": "Rhode Island"
+            "South Carolina": "South Carolina"
+            "South Dakota": "South Dakota"
+            "Tennessee": "Tennessee"
+            "Texas": "Texas"
+            "Utah": "Utah"
+            "Vermont": "Vermont"
+            "Virgin Islands": "Virgin Islands"
+            "Virginia": "Virginia"
+            "Washington": "Washington"
+            "West Virginia": "West Virginia"
+            "Wisconsin": "Wisconsin"
+            "Wyoming": "Wyoming"
+    - fill_in:
+        - name: "dnn7787ZipCode"
+          selector: "#dnn7787ZipCode"
+          value: $ADDRESS_ZIP5
+          required: true
+        - name: "dnn7787Phone"
+          selector: "#dnn7787Phone"
+          value: $PHONE
+          required: true
+        - name: "dnn7787Comments"
+          selector: "#dnn7787Comments"
+          value: $MESSAGE
+    - check:
+        - name: "dnn7787RespondtoOpinion"
+          selector: "#dnn7787RespondtoOpinion-0"
+          value: "Yes"
+          required: true
+    - click_on:
+        - value: "Send Now"
+          selector: "#dnn7787SendNow"
+    # Wait for the "processing" button to go to the next page before looking for the success criteria.
+    - wait:
```

Comment:
Curious if this is necessary? By default we'll wait for 10 seconds in any case.

Reply:
Removed.